docker/demo/compose.simulation.yaml

```diff
@@ -19,4 +22,6 @@ services:
     volumes:
       - /tmp/.X11-unix:/tmp/.X11-unix:rw
     command: >
-      ros2 launch panther_gazebo simulation.launch.py robot_model:=panther namespace:=panther
+      ros2 launch panther_gazebo simulation.launch.py
+        namespace:=panther
+        robot_model:=panther
```

Comment:
Personally, I stopped being a fan of this solution when I had more than 5 exports to autonomy. Then we add a way of configuring it, which should be listed. As far as I'm concerned, we have a table with arguments for this, so that everyone can look at it and add their own settings if necessary.
I added what you asked for, because in this case it is written in the readme that it is based on envie, so it makes sense